pages/querying/vector-search.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Vector search
+description: Learn how to use vector search and manage vector indices in Memgraph.
+---
+
+import { Callout } from 'nextra/components'
+
+# Vector search
+
+The vector search is currently an [experimental
+feature](/database-management/experimental-features). 
+
+## Run Memgraph with vector search feature 
+
+To try out vector search feature you need to **enable** it and **configure** it.
+To enable the feature set the `--experimental-enabled` to `vector-search`. To
+configure the feature set the `--experimental-config` flag. 
+
+<Callout type="info">
+Changing the configuration settings depends on the way you are using Memgraph,
+so please refer to the [configuration
+docs](/database-management/configuration#changing-configuration) for more
+information.
+</Callout>
+
+Here is the example configuration:
+
+```shell
+--experimental-config=
+'
+{
+  "vector-search": {
+    "index_name": {
+      "label": "Node",
+      "property": "vector",
+      "dimension": 2,
+      "capacity": 1000,
+      "metric": "cos",
+      "resize_coefficient": 2,
+    }
+  }
+}
+'
+```
+
+Keep in mind that `--experimental-enabled` and `--experimental-config` flags are
+both required and the following fields are mandatory for the configuration:
+`label`, `property`, `dimension` and `capacity`.
+
+## Metric kinds
+
+The following table lists the supported metric kinds for vector search. These
+metrics determine how distances between vectors are calculated. Default type
+for the metric is `l2sq`.
+
+| Metric      | Description                                          |
+|-------------|------------------------------------------------------|
+| `ip`        | Inner product (dot product).                         |
+| `cos`       | Cosine similarity.                                   |
+| `l2sq`      | Squared Euclidean distance.                          |
+| `pearson`   | Pearson correlation coefficient.                     |
+| `haversine` | Haversine distance (suitable for geographic data).   |
+| `divergence`| A divergence-based metric.                           |
+| `hamming`   | Hamming distance.                                    |
+| `tanimoto`  | Tanimoto coefficient.                                |
+| `sorensen`  | Sørensen-Dice coefficient.                           |
+| `jaccard`   | Jaccard index.                                       |
+
+## Scalar kinds
```

Comment:
Again, I don't like kinds word here if it's not a vector thing. Type?

Reply:
I agree that using the term "type" here (and for the metric as well) is more intuitive. It’s not specific to vectors, so "type" works. I only used "kind" because that’s how it’s documented in the `usearch` docs.
Do we want to use "type" instead then?

Reply:
Changed to type for now.